test/boost/view_build_test.cc

```diff
@@ -74,7 +74,7 @@ SEASTAR_TEST_CASE(test_builder_with_large_partition) {
                       "primary key (v, c, p)").get();
 
         f.get();
-        auto built = e.get_system_keyspace().load_built_views().get0();
+        auto built = e.get_system_keyspace().local().load_built_views().get0();
```

Comment:
nit: we could also introduce a `get_local_system_keyspace()`, not that it is much shorter.